packages/calcite-components/src/components/modal/modal.tsx

```diff
@@ -398,20 +397,6 @@ export class Modal
 
   @State() defaultMessages: ModalMessages;
 
-  //--------------------------------------------------------------------------
-  //
-  //  Event Listeners
-  //
-  //--------------------------------------------------------------------------
-
-  @Listen("keydown", { target: "window" })
-  handleEscape(event: KeyboardEvent): void {
```

Comment:
FIXME: We need to keep this, or something similar, to preserve closing with `Escape` when developers set `focusTrapDisabled = true`. When `true`, focus traps are no longer active and therefore won't handle `Escape` presses. This would also apply to anything using `clickOutsideDeactivates`.